.github/workflows/ci.yaml

```diff
@@ -16,15 +16,32 @@ permissions:
   contents: read
 
 jobs:
+    # Dynamically generate a bit of JSON depending on whether our action has access to repository secrets.
```

Comment:
Need to revert that line above that changed to pull_request_target too

Reply:
Updated. I assuming keeping the `permissions` is ok?

Reply:
Yeah I don't think that matters, though I'd remove just to keep things more uniform